packages/commonwealth/main.ts

```diff
@@ -151,30 +165,38 @@ export async function main(app: express.Express) {
     app.use(favicon(`${__dirname}/favicon.ico`));
     app.use('/static', express.static('static'));
 
-    app.use(
-      pinoHttp({
-        quietReqLogger: false,
-        transport: {
-          target: 'pino-http-print',
-          options: {
-            destination: 1,
-            all: false,
-            colorize: true,
-            relativeUrl: true,
-            translateTime: 'HH:MM:ss.l',
+    if (!testing) {
```

Comment:
We should do this with a LOGGING_ENV var rather than this. If we want to remove logging during tests by default we can but we should also be able to enable logging for tests without having to modify all main() calls.

Reply:
Just trying to get all tests passing. Will reorganize the options in the next commit, so we can control the app features from the outside

Reply:
@timolegros commit [fdce6e0](https://github.com/hicommonwealth/commonwealth/pull/7220/commits/fdce6e01c310d353442b0d9588caef9824cbe072) moves most .env details and adapters to the entry point (server or test-server), trying to localize the configuration outside the main setup function. We can define more logging env vars for testing in future PRs if needed.

Reply:
Could we just add it as part of this PR? Otherwise I will literally create a PR rn anyways. Being able to get logs in tests is crucial for debugging.

Reply:
added TEST_WITH_LOGS env var